File: ESC.Resturant.Data/Entities/BoxMonyCategory.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace ESC.Resturant.Data.Entities
{
    public class BoxMonyCategories
    {

        [Key]
        public int Id { get; set; }
        public string NameAR { get; set; }
        public string NameEN { get; set; }
        public bool IsDelete { get; set; }
        public bool IsCredit { get; set; }
        public bool IsDebit { get; set; }
        public virtual ICollection<BoxMonies> BoxMonies { get; set; }
    }
}

File: ESAResturant.NEW.web/ClientApp/src/app/admin/BoxMony/Models/boxMonyCategory.ts
import { boxMony } from '../Models/boxMony';


export class BoxMonyCategory {

  constructor() {

    this.id = 0;
    this.nameAR = "";
    this.nameEN = "";
    this.isDelete = false;
    this.isCredit = false;
    this.isDebit = false;
    //this.boxMony = new boxMony();
  }


  id: number;
  nameAR: string;
  nameEN: string;
  isDelete: boolean;
  isCredit: boolean;
  isDebit: boolean;
  boxMony: boxMony;
}
